host-configs/LLNL/quartz-base.cmake

```diff
@@ -58,5 +58,12 @@ set(MKL_LIBRARIES ${MKL_ROOT}/lib/intel64/libmkl_intel_lp64.so
 
 # ATS
 set(ATS_ARGUMENTS "--machine slurm36"  CACHE STRING "")
+set(USER $ENV{USER} CACHE STRING "")
+set(ATS_WORKING_DIR "/p/lustre2/${USER}/integratedTests/${CONFIG_NAME}"  CACHE PATH "")
+set(ATS_BASELINE_DIR "/p/lustre2/${USER}/integratedTests/baselines"  CACHE PATH "")
+
+# Temporary argument for python module change testing
+set(GEOS_PYTHON_PACKAGES_BRANCH "feature/sherman/outOfPlaceATS" CACHE STRING "" FORCE)
```

Comment:
This is a cmake variable that can override the branch of geosPythonPackages during the build.  We'll want to have a procedure in place for handling this when we make potentially breaking changes to the CI

Reply:
makes sense. We'll need that cmake target to reinstall the packages though.

Reply:
Is it also working if we put and hash?

Reply:
That shouldn't be hard to hard to do